.gitignore

```diff
@@ -396,3 +396,4 @@ FodyWeavers.xsd
 
 # JetBrains Rider
 *.sln.iml
+src/InstagramClone.Api/InstagramClone.Api.csproj
```

Comment:
Why are you adding your csproj project file to gitignore?

Reply:
Oops, what was I supposed to have done?

Reply:
so if you can explain why you have this line here. 

Did you put it there yourself or was it a mistake?

Reply:
I right-clicked on it when I was about to commit from git desktop then git ignored the file, that was what happened

Reply:
Okay noted.

However this file is a very important file and your project can't w0rk without it. You cannot afford to ignore this for source control purposes, you should remove it from gitignore.

You can read more about the file here.

https://docs.microsoft.com/en-us/aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/understanding-the-project-file